src/WebJobs.Script/Diagnostics/FileTraceWriter.cs

```diff
@@ -153,7 +153,17 @@ public override void Trace(TraceEvent traceEvent)
                 return;
             }
 
-            AppendLine($"[{traceEvent.Level}] {traceEvent.Message}");
+            // format the trace line metadata prefix
+            var traceProperties = new List<string>()
+            {
+                traceEvent.Level.ToString()
+            };
+            if (traceEvent.Properties.TryGetValue(ScriptConstants.TracePropertyFunctionNameKey, out value))
+            {
+                traceProperties.Add((string)value);
+            }
+            string tracePrefix = string.Join(",", traceProperties);
```

Comment:
I realize however that this will also add FunctionName to function specific log files which is unecessarily redundant. We only want to do that for host logs I think. I can make a change so it is only added for host logs.